src/java.base/share/classes/java/lang/constant/MethodHandleDesc.java

```diff
@@ -205,6 +205,9 @@ default MethodHandleDesc asType(MethodTypeDesc type) {
      */
     MethodTypeDesc invocationType();
 
+    /**
+     * @since 21
```

Comment:
```suggestion
     * {@inheritDoc}
     *
     * @since 21
```

Reply:
Will need to look more into what elements require `{@inheritDoc}`